From bc2565c425537df465698819bb5786252d10b309 Mon Sep 17 00:00:00 2001
From: Jukka Laitinen <jukka.laitinen@intel.com>
Date: Wed, 22 Mar 2017 19:08:22 +0200
Subject: [PATCH] patch

Signed-off-by: Jukka Laitinen <jukka.laitinen@intel.com>
---
 FindOpenCL.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/FindOpenCL.cmake b/FindOpenCL.cmake
index 9810dd2..dc27dbf 100644
--- a/FindOpenCL.cmake
+++ b/FindOpenCL.cmake
@@ -109,7 +109,7 @@ endif( )
 
 if( LIB64 )
   find_library( OPENCL_LIBRARIES
-    NAMES OpenCL
+    NAMES OpenCL cl
     HINTS
       ${OPENCL_ROOT}/lib
       $ENV{OPENCL_ROOT}/lib
@@ -119,6 +119,7 @@ if( LIB64 )
     PATH_SUFFIXES x86_64 x64 x86_64/sdk
     PATHS
     /usr/lib
+    /usr/lib/beignet-570x
     /usr/local/cuda/lib
   )
 else( )
-- 
2.7.4

